generators/client/templates/react/src/main/webapp/app/modules/account/password-reset/password-reset.reducer.ts.ejs

```diff
@@ -73,7 +73,7 @@ const apiUrl = '/api/account/reset-password';
 export const handlePasswordResetInit = mail => ({
   type: ACTION_TYPES.RESET_PASSWORD_INIT,
   // If the content-type isn't set that way, axios will try to encode the body and thus modify the data sent to the server.
-  payload: axios.post(`${apiUrl}/init`, mail, { headers: {['Content-Type']: 'text/plain'}}),
+  payload: axios.post(`${apiUrl}/init`, mail, { headers: {'Content-Type': 'text/plain'}}),
```

Comment:
see the comment above the line, the content-type was set this way for a reason, so please test carefully if changing it